pass-manager-svc/src/testFixtures/kotlin/com/example/passmanagersvc/util/PassProtoFixture.kt

```diff
@@ -0,0 +1,126 @@
+package com.example.passmanagersvc.util
+
+import com.example.internal.commonmodels.Error
+import com.example.internal.input.reqreply.CancelPassRequest
+import com.example.internal.input.reqreply.CancelPassResponse
+import com.example.internal.input.reqreply.CreatePassRequest
+import com.example.internal.input.reqreply.CreatePassResponse
+import com.example.internal.input.reqreply.DeletePassByIdRequest
+import com.example.internal.input.reqreply.DeletePassByIdResponse
+import com.example.internal.input.reqreply.FindPassByIdRequest
+import com.example.internal.input.reqreply.FindPassByIdResponse
+import com.example.internal.input.reqreply.TransferPassRequest
+import com.example.internal.input.reqreply.TransferPassResponse
+import com.example.passmanagersvc.domain.MongoPass
+import com.example.passmanagersvc.util.PassFixture.passToCreate
+import com.example.passmanagersvc.web.mapper.proto.DecimalProtoMapper.toBDecimal
+import com.example.passmanagersvc.web.mapper.proto.pass.FindPassByIdMapper.toProto
+import org.bson.types.ObjectId
+
+object PassProtoFixture {
+    private const val PASS_OWNER_NOT_FOUND = "Could not find pass owner by id "
+    private const val PASS_NOT_FOUND = "Could not find pass by id "
+    private const val PASS_TYPE_NOT_FOUND = "Could not find pass type by id "
+
+    fun cancelPassRequest(passId: ObjectId?, passOwnerId: ObjectId?): CancelPassRequest {
+        return CancelPassRequest.newBuilder()
+            .setId(passId.toString())
+            .setOwnerId(passOwnerId.toString())
+            .build()
+    }
```

Comment:
1. should incoming parameters be nullable?
2. be careful with setting nullable fields to proto classes. you will receive NPE